crates/oxc_linter/src/snapshots/eslint_no_debugger.snap

```diff
@@ -6,4 +6,4 @@ source: crates/oxc_linter/src/tester.rs
  1 │ if (foo) debugger
    ·          ────────
    ╰────
-  help: Replace `debugger` with `{}`.
+  help: Replace the debugger statement with an empty block
```

Comment:
For me, a block is surrounded by `{}`. Would prefer just the `REMOVE_DEBUGGER` message. 

Can add later a test for https://github.com/oxc-project/oxc/issues/9561 to find different messages